float32 x
float32 y
float32 z
float32 psi 
float32 theta 
float32 phi 
---
float32[6] motor_angles